inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -113,16 +113,18 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
   :root {
     --bslib-dashboard-sidebar-bg: var(--#{$prefix}body-bg);
     --bslib-dashboard-sidebar-main-bg: var(--#{$prefix}body-bg);
+    --bslib-dashboard-main-bg: rgb(247, 247, 247);
```

Comment:
We're losing the ability to have this color flow nicely from `--bs-emphasis-color-rgb`, but we gain the fact that dashboard regions aren't additive (not that that's a big gain). More importantly, this lets us correctly set the background color via the `<body>` tag on all browsers.